examples/redis-operator.md

```diff
@@ -0,0 +1,115 @@
+# Redis-pmem operator
+This readme describes a complete example to deploy a Redis cluster through the [redis-operator](https://github.com/spotahome/redis-operator) using QEMU-emulated persistent memory devices.
+
+## Prerequisites:
+* Docker-ce: version 18.06.1
+* [Docker registry](https://docs.docker.com/registry/deploying/).
+
+## PMEM-CSI installation
+The steps below describe how to install PMEM-CSI within a K8S cluster that contains one master and three worker nodes which use a specific version of [Clear Linux OS](https://clearlinux.org/) for the nodes, so the following steps can be reproduced.
+1. Clone this project into `$GOPATH/src/github.com/intel`.
+2. Build PMEM-CSI:
+    ```sh
+     $ cd $GOPATH/src/github.com/intel/pmem-CSI
+     $ make push-images
+    ```
+    **Note**: by default the build images are going to be pushed into a local [Docker registry](https://docs.docker.com/registry/deploying/).
```

Comment:
Change to: 
By default, the build images are pushed

Reply:
Done.